go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/containerd/go-runc v1.0.0
 	github.com/containerd/imgcrypt v1.1.4
 	github.com/containerd/log v0.1.0
-	github.com/containerd/nri v0.1.0
+	github.com/containerd/nri v0.1.1-0.20231113145513-51b5b48e4e21 // release/0.1 branch
```

Comment:
Using head of the branch for now, but I'll update this to use a tag once created.